setup.md

```diff
@@ -34,10 +44,10 @@ driver = wd
 ```js
 const assert = require('assert');
 
-const browser = require('testium-mocha').browser;
+const { browser } = require('testium-mocha');
 
 describe('The homepage', () => {
-  before(browser.beforeHook());
+  before(browser.beforeHook({ driver: 'wd' }));
```

Comment:
maybe prepend comment about how you don't need `{ driver: 'wd' }` if you have it in your `.testiumrc`